src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -804,27 +833,29 @@ public static implicit operator (int a, int b)(NewStruct value)
     }
 }
 ";
-            await TestInRegularAndScriptAsync(text, expected, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+            await Test(text, expected, options: PreferImplicitTypeWithInfo(), testHost: host);
         }
 
         [Theory, CombinatorialData, Trait(Traits.Feature, Traits.Features.CodeActionsConvertTupleToStruct)]
         public async Task TestNonLiteralNames(TestHost host)
         {
             var text = @"
+using System.Collections.Generic;
```

Comment:
📝 These `using` directives should not be added as part of this PR. The change hides a design limitation in the refactoring.

Reply:
have split these into tests that do both forms.